src/status_im/ui/screens/wallet/request/views.cljs

```diff
@@ -45,7 +46,9 @@
                                      :input-options {:default-value  (str (money/to-fixed (money/wei->ether amount)))
                                                      :max-length     21
                                                      :on-focus       (fn [] (when @scroll (utils/set-timeout #(.scrollToEnd @scroll) 100)))
-                                                     :on-change-text #(re-frame/dispatch [:wallet.request/set-and-validate-amount %])}}]]]
+                                                     :on-change-text #(reset! amount-text-value %)
+                                                     :on-end-editing #(re-frame/dispatch [:wallet.request/set-and-validate-amount @amount-text-value])
+                                                     }}]]]
```

Comment:
@ovx can you please place all trailing parentheses on a single line instead of distinct lines?

we are following this guide https://github.com/bbatsov/clojure-style-guide#gather-trailing-parens